charts/node-analyzer/templates/_helpers.tpl

```diff
@@ -222,8 +229,15 @@ nodeAnalyzer agentConfigmapName
     {{- default .Values.global.agentConfigmapName | default "sysdig-agent" -}}
 {{- end -}}
 
-{{- define "nodeAnalyzer.deployHostAnalyzer" -}}
```

Comment:
I assume nodeAnalyzer.deployHostAnalyzer is being removed and changed for nodeAnalyzer.deployHostscanner if that is the case maybe we can remove nodeAnalyzer.hostAnalyzer.deploy from the readme.
Edit: change not needed as it is present below.

Reply:
HostAnalyzer and HostScanner are two different components, so both shall remain :)